packages/cubejs-schema-compiler/test/integration/postgres/sql-generation.test.ts

```diff
@@ -976,12 +976,12 @@ SELECT 1 AS revenue,  cast('2024-01-01' AS timestamp) as time UNION ALL
     timeDimensions: [
       {
         dimension: 'visitors.created_at',
-        granularity: 'three_days',
```

Comment:
Originally, this test checks for custom granularities in rolling window, but you changed it to predefined one. Not a great deal. You can add another test with 2 predefined granularities, but please don't change existing tests to reduce the test cases.

Reply:
I agree it makes sense to bring it back. Although to be fair, that test was actually testing two time_dimensions, not custom granularities. I think the best approach would be to keep my change, and move the old test to custom-granularity. It’s generally better when a test focuses on a single piece of functionality instead of mixing multiple concerns.